test/com/facebook/buck/features/ocaml/OCamlIntegrationTest.java

```diff
@@ -517,8 +517,8 @@ public void testConfigWarningsFlags() throws IOException {
     workspace.runBuckCommand("build", target.toString()).assertFailure();
     BuckBuildLog buildLog = workspace.getBuildLog();
     assertTrue(buildLog.getAllTargets().containsAll(targets));
-    buildLog.assertTargetCanceled(target);
-    buildLog.assertTargetCanceled(binary);
+    buildLog.assertTargetFailed(target.toString());
```

Comment:
What's the difference between canceled and failed?

Reply:
There is no doc defines them and assertTargetCanceled() is not used anywhere in buck code. My understanding is CANCELED is used internally and the final result in buildLog is always SUCCESS or FAILED.

Reply:
All ocaml tests are skipped in Sandcastle.